iroh-resolver/src/unixfs_builder.rs

```diff
@@ -954,6 +954,21 @@ mod tests {
         Ok(())
     }
 
+    #[tokio::test]
+    async fn symlink_from_disk_test() -> Result<()> {
+        let temp_dir = std::env::temp_dir();
```

Comment:
while this works, I would recommend using https://docs.rs/tempfile it is more reliable and takes care of cleanup